pkgs/applications/networking/cluster/kube3d/default.nix

```diff
@@ -22,11 +22,13 @@ buildGoModule rec {
   '';
 
   nativeBuildInputs = [ installShellFiles ];
+
+  # TODO: Move to enhanced installShellCompletion when in master: PR #83630
```

Comment:
Isn't #83630 already merged?

Reply:
It's not in master yet
![image](https://user-images.githubusercontent.com/9866621/97157780-76181d80-1770-11eb-91dd-4865c713e065.png)

Reply:
right, my bad, it's only on staging